final-project/static/scripts.js

```diff
@@ -211,6 +213,58 @@ function displayProduct(jsonOfProductInfo) {
 };
 
 // TODO: Render "edit product info" UI.
+function editProduct() {
+
+  // TODO: Create form elements for all the required page elements.
+  // Submit the form via the POST method.
+
+  // Select all the product elements (could be own function).
+  // It is used in the displayProduct code as well.
+
+  // Select product name.
+  var prodPageName = document.querySelector('.product-name');
```

Comment:
You can also reference elements by ID so that you know you are access the specific element you want.

Reply:
Got it. I will start to use structural naming conventions. https://www.webpagefx.com/blog/web-design/css-tip-2-structural-naming-convention-in-css/